python/ElectronsProducer.py

```diff
@@ -5,38 +5,31 @@
         prefix = cms.string('electron_'),
         enable = cms.bool(True),
         parameters = cms.PSet(
-            src = cms.untracked.InputTag('slimmedElectrons'),
+            src = cms.untracked.InputTag('phase2Electrons'),
             ea_R03 = cms.untracked.FileInPath('RecoEgamma/ElectronIdentification/data/Summer16/effAreaElectrons_cone03_pfNeuHadronsAndPhotons_80X.txt'),
             ea_R04 = cms.untracked.FileInPath('cp3_llbb/Framework/data/effAreaElectrons_cone04_pfNeuHadronsAndPhotons.txt'),
             ids = cms.untracked.VInputTag(
-                'egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-veto',
-                'egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-loose',
-                'egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-medium',
-                'egmGsfElectronIDs:cutBasedElectronID-Summer16-80X-V1-tight',
-                'egmGsfElectronIDs:cutBasedElectronHLTPreselection-Summer16-V1',
-                'egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp90',
-                'egmGsfElectronIDs:mvaEleID-Spring16-GeneralPurpose-V1-wp80',
-                'egmGsfElectronIDs:mvaEleID-Spring16-HZZ-V1-wpLoose'
+#                'No ids defined for phase2'
                 ),
-            mva_id = cms.untracked.PSet(
-                values = cms.untracked.InputTag('electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16GeneralPurposeV1Values'),
-                categories = cms.untracked.InputTag('electronMVAValueMapProducer:ElectronMVAEstimatorRun2Spring16GeneralPurposeV1Categories')
-            ),
+#            mva_id = cms.untracked.PSet(
```

Comment:
Mostly a matter of style, but there's no need to keep these around commented, it only adds confusion (for the mva_id it could be argued that they show the structure, but the whole list of scalefactors is definitely not needed)